update/application.go

```diff
@@ -81,6 +84,12 @@ type deployJob struct {
 	Timeout *time.Duration `help:"Timeout of the job." placeholder:"${app_default_deploy_job_timeout}"`
 }
 
+type workerJob struct {
+	Name    *string `help:"Name of the worker job to add." placeholder:"worker-1"`
```

Comment:
Do all the fields need to be `*string`? Wouldn't non pointer strings be sufficient and easier to handle?

Reply:
Ok, I understand. 👍